src/main/java/gift/repository/WishRepository.java

```diff
@@ -0,0 +1,53 @@
+package gift.repository;
+
+import gift.entity.Wish;
+import gift.dto.WishResponseDto;
+import org.springframework.jdbc.core.simple.JdbcClient;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+@Repository
+public class WishRepository {
+
+    private final JdbcClient jdbcClient;
+    private final SimpleJdbcInsert jdbcInsert;
+
+    public WishRepository(JdbcClient jdbcClient, DataSource dataSource) {
+        this.jdbcClient = jdbcClient;
+        this.jdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("wish")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    public void save(Long memberId, Long productId) {
+        var params = new java.util.HashMap<String, Object>();
```

Comment:
HashMap을 Import해서 경로는 생략하도록하는게 조금 더 깔끔하지 않을까요? 

```suggestion
        var params = new HashMap<String, Object>();
```